ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/mainnet/MainnetBlockBodyValidator.java

```diff
@@ -224,6 +224,17 @@ private boolean isOmmerValid(
       return false;
     }
 
+    if (ommerValidationMode != HeaderValidationMode.LIGHT
+        && ommerValidationMode != HeaderValidationMode.LIGHT_SKIP_DETACHED
+        && ommerValidationMode != HeaderValidationMode.LIGHT_DETACHED_ONLY) {
```

Comment:
Could this be clarified with something like:  `!HeaderValidationMode.isLight(ommerValidationMode)` ?